master/static/migrations/20220512160917_update-job-type-jobs.tx.down.sql

```diff
@@ -0,0 +1,10 @@
+ALTER TYPE public.job_type RENAME TO _job_type; 
+CREATE TYPE public.job_type AS ENUM (
+    'EXPERIMENT',
+    'NOTEBOOK',
+    'SHELL',
+    'COMMAND',
+    'TENSORBOARD'
+);
+ALTER TABLE public.jobs ALTER COLUMN job_type TYPE public.job_type USING (job_type::text::job_type);
+DROP TYPE _job_type;
```

Comment:
this migration isn't necessarily safe to run if we ran on this version for a while. probably just run a `DELETE FROM public.jobs WHERE job_type = 'CHECKPOINT_GC'` first or something.